src/components/Navbar/NavTabs.js

```diff
@@ -113,6 +115,26 @@ function NavTabs() {
     ))
   }
 
+
+  function handleFirstTab(e) {
+    if (e.keyCode === 9) {
+      document.body.classList.add("user-is-tabbing")
+
+      window.removeEventListener("keydown", handleFirstTab)
```

Comment:
![image](https://user-images.githubusercontent.com/19617248/94065702-7f048080-fdb9-11ea-8bfe-9fd8b0f11650.png)

- this causes the build to fail, can you implement the fix that @gh0stl0nely did before for this issue?

Reply:
Fixed.